pkcs11/CMakeLists.txt

```diff
@@ -75,6 +86,7 @@ target_link_libraries(
   yubihsm_pkcs11
   yubihsm
   ${LIBCRYPTO_LDFLAGS}
+  ${GETOPT_LIBS}
```

Comment:
what's this? do we need to link with a special getopt lib?

Reply:
Yes. 

gengetopt is not available on Windows. Before, we used to use gengetopt binaries compiled with mingw, but this does not work with MSVC. So instead, cmdline.c/h are generated on a nix system and builds on Windows use the source release package and not the source directly cloned from the repository. getopt.dll is the "getopt"-part that's available with MSVC. It will be included in the SDK package along with its license.